fireapp/pubspec.lock

```diff
@@ -225,14 +225,6 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "2.3.2"
-  dbus:
```

Comment:
The fact all these dependencies were removed and the flutter sdk lowered from 3.16.0 to 3.7.0 should tell you something went wrong with the lock. Please ensure you are using `fvm` when running flutter commands, do not just click on the IDE option (it will use whatever flutter you have on your system).